aggregator_server/aggregator_server.cpp

```diff
@@ -68,11 +69,8 @@ using napi_handler = libzeth::groth16_api_handler<npp>;
 
 using nsnark = typename nverifier::snark;
 
-static const size_t batch_size = 2;
-static const size_t num_inputs_per_nested_proof = 1;
-
-using aggregator_circuit =
-    libzecale::aggregator_circuit<wpp, wsnark, nverifier, batch_size>;
+using aggregator_circuit = libzecale::
+    aggregator_circuit<wpp, wsnark, nverifier, libzecale::ZECALE_BATCH_SIZE>;
```

Comment:
Could be worth doing:
```
static const size_t batch_size = libzecale::ZECALE_BATCH_SIZE;
static const size_t num_inputs_per_nested_proof = libzecale::ZECALE_NUM_INPUTS_PER_NESTED_PROOF);
```
to avoid all the long references to the config variable below.  (Not critical, so feel free to skip).